### PR TITLE
[3158] Update ruby version to 2.6.5 and nodejs to 12.16.1

### PR DESCRIPTION
**Pull request by AgaDufrat**
### Guidance to review
- `asdf install ruby 2.6.5`
- `asdf install nodejs 12.16.1`
- `bundle install`
- check if the app and tests are running 

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product review

### CODE REVIEW

**Comment by AgaDufrat** [comment]
[Update Ruby everywhere to at least 2.6.5](https://trello.com/c/tvZnwfLX/3158-update-ruby-everywhere-to-at-least-265)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.1-alpine
+FROM ruby:2.6.5-alpine
```
Comment (aliuk2012) [inline Dockerfile]: When I looked at updating node, I noticed that 2.6.5 image uses node 12.

Comment (aliuk2012) [reply Dockerfile]: I think we should try to ensure that we are using the same version of node between what we are deploying and using locally. Perhaps we should bump node version in this pr

Comment (AgaDufrat) [reply Dockerfile]: sure, will give it a go

**Comment by aliuk2012** [comment]
Any thoughts about going straight to Ruby 2.7.0? To soon? I attempted it locally and the tests pass but a few deprecation warnings being raised by some of the gems we are using.

**Pull request merged**